homeassistant/components/fluss/__init__.py

```diff
@@ -0,0 +1,55 @@
+"""The Fluss+ integration."""
+
+from __future__ import annotations
+
+import logging
+
+from fluss_api import (
+    FlussApiClient,
+    FlussApiClientAuthenticationError,
+    FlussApiClientCommunicationError,
+    FlussApiClientError,
+)
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+
+from .const import CONF_API_KEY, CONF_BASE_URL, DOMAIN  # noqa: F811
+
+LOGGER = logging.getLogger(__package__)
+
+PLATFORMS: list[Platform] = [Platform.BUTTON]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Fluss+ from a config entry."""
+
+    try:
+        api = FlussApiClient(
+            entry.data[CONF_API_KEY],
+            entry.data.get(
+                CONF_BASE_URL,
+                "https://zgekzokxrl.execute-api.eu-west-1.amazonaws.com/v1/api/",
```

Comment:
this is not set in the config flow, so this will always be `None`. The URL itself should be in the library